app/scripts/controllers/transactions.js

```diff
@@ -26,7 +26,7 @@ angular.module('openhimConsoleApp')
     $scope.settings.list = {};
     $scope.settings.list.tabview = 'same';
     $scope.settings.filter = {};
-    $scope.settings.filter.limit = 10;
+    $scope.settings.filter.limit = 100;
```

Comment:
20 could be good? Unlike 100 it wouldn't take much to scroll down to notice the buttons at the end. Maybe we could also add a tooltip to the limit label at the top saying something like "tip: you can change your default limit in your user profile"?